scripts/upgrade

```diff
@@ -205,6 +206,16 @@ chmod 750 "$datadir"
 chmod -R o-rwx "$datadir"
 chown -R $app:www-data "$datadir"
 
+#=================================================
+# Generate secrets if they don't exist
+#=================================================
+
+if [ -z "$secrets_peertube" ]; then
+	ynh_script_progression --message="Generating and storing PeerTube secrets..."
```

Comment:
Minor thing but I would tend to have an `ynh_print_info` here. Otherwise, because of the way `ynh_script_progression` works and because it's inside an `if`, it's going to create funky stuff where the progress bar is gonna reach like 120% or never reach 100% ...

```suggestion
	ynh_print_info --message="Generating and storing PeerTube secrets..."
```

Reply:
Done! Thanks